content/Video_Display.md

```diff
@@ -563,21 +563,21 @@ Whether the background is displayed can be toggled using
 [LCDC bit 0](#lcdc-0-bg-window-display-priority), except on CGB in CGB Mode,
 where it's always drawn.
 
-### The Window
-
-Besides background, there is also a "window" overlaying the
-background. The window is not scrollable, that is, it is always
-displayed starting from its left upper corner. The location of a window
-on the screen can be adjusted via WX and WY registers. Screen
-coordinates of the top left corner of a window are WX-7,WY. The tiles
-for the window are stored in the Tile Data Table. Both the Background
-and the window share the same Tile Data Table.
-
-Whether the window is displayed can be toggled using
-[LCDC bit 5](#lcdc-5-window-display-enable). Enabling the window makes
+### Window
+
+Besides the Background, there is also the Window overlaying the
```

Comment:
"Besides the Background, there is also the Window overlaying the Background."
I don't think this sounds better

Reply:
Fixed